source/extensions/filters/network/well_known_names.h

```diff
@@ -28,8 +28,8 @@ class NetworkFilterNameValues {
   const std::string TCP_PROXY = "envoy.tcp_proxy";
   // Authorization filter
   const std::string EXT_AUTHORIZATION = "envoy.ext_authz";
-  // Thrift filter
-  const std::string THRIFT = "envoy.thrift";
+  // Thrift proxy filter
+  const std::string THRIFT_PROXY = "envoy.thrift_proxy";
```

Comment:
new filters should be `envoy.filters.network.name`... (per the comment above)

Reply:
Oops. missed that. Thanks.